### ISSUE TITLE
Datasource Value Set - what is the purpose?

### ISSUE THREAD

**Issue by larrybabb** [clingen-data-model/clingen-interpretation#93]
The InformationSource (frmly known as DataSource) has an evidenceSourceType attribute which is a class/type/category of the resource or source from where a particular item of information originated.

While I think it is very useful to capture the Information Source itself, I've lost track of why we need to classify the types of sources themselves.

Right now we have
lit - Literature
db - Database
app - Application

While this seems interesting, I can't think why we are doing it and therefore cannot figure out how to determine what the scope, level or constraints by which this set of values should be defined.

If we cannot concretely describe the purpose of value. I suggest we drop this.

Thoughts?

**Comment by rrfreimuth** [comment]
I agree that we should drop this if there is no driving reason.  Modeling can go on forever so at some point we need to put bounds around the problem.

The only reason I can think of that would justify doing this is if we anticipated that we would need to separate evidence by source type.  For example, if we wanted to weight evidence from literature stronger than those from a different source, or if we wanted to provide the ability for users to query by source type.

Furthermore, I would guess that there will be a finite number of values used for InformationSource.  If at some point in the future we wanted to normalize evidenceSourceType, it should be feasible to do.

**Comment by bpow** [comment]
With `E36` having been dropped, the build was broken because there was still a markdown file referring to it (`source/tech/details/evidence_data-source.md`).

I removed the relevant file in f2e59ffb0d9 and am mostly just commenting here in case I should have done something else (thoughts, @cbizon ?).

**Comment by cbizon** [comment]
Not really.  Having to manually remove the md page is maybe pointing to a gross part of the build process (having structures defined in both the sheet and the md).  We could probably insert the text in there into the spreadsheet, and then generate the md on the fly, if we were so inclined.
